src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const robotKeys = Object.values(robot);
+  const robotValues = Object.keys(robot);
+
+  for (let index = 0; index < robotKeys.length; index++) {
```

Comment:
use `for in` loop to iterate robot object, in that case robotKeys and robotValues are redundant